src/main/java/service/FruitTransactionServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package service;
+
+import dao.FruitTransactionDao;
+import java.util.List;
+import model.FruitTransaction;
+
+public class FruitTransactionServiceImpl implements FruitTransactionService {
+    private final FruitTransactionDao fruitTransactionDao;
+
+    public FruitTransactionServiceImpl(FruitTransactionDao fruitTransactionDao) {
+        this.fruitTransactionDao = fruitTransactionDao;
+    }
+
+    @Override
+    public void createNewFruitTransaction(List<String[]> fruitsInputDataList) {

```

Comment:
Lets make this method to return List< FruitTransaction> (we dont need to store FruitTransactions in Storage)